protos/flyteidl/plugins/mpi.proto

```diff
@@ -0,0 +1,14 @@
+syntax = "proto3";
+
+package flyteidl.plugins;
+
+option go_package = "github.com/flyteorg/flyteidl/gen/pb-go/flyteidl/plugins";
+
+// Custom proto for plugin that enables distributed training using https://github.com/kubeflow/mpi-operator
+message DistributedMPITrainingTask {
+  // number of worker,launcher replicas spawned in the cluster for this job
```

Comment:
you need to add a comment to each field, that will automatically generate in the docs